docs/introduction/v1-and-v2.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: /versions
+---
+
+# v1 and v2
+
+There are two versions of the Friendly Captcha service, called **v1** and **v2**. The second version contains technical and non-technical enhancements that come from state-of-the-art improvements and learnings from **v1**.
+
+## What's new in v2?
+
+* v2 has **improved protection**. It provides us with more powerful signals to detect abuse, automated browsers, and browsers that have otherwise been tampered with.
+* v2 improves on **user experience**. It has an updated design and the ability to store solutions for future submissions.
+* v2 has a **smaller speed difference** between older and newer browsers.
```

Comment:
```suggestion
```

I guess the reader of this page doesn't care about this that much. It's an advantage for v2, but I guess it can be behind the scenes.